### ISSUE TITLE
Add example project

### ISSUE THREAD

**Issue by PaulRBerg** [lampshade9909/DiamondSetter#4]
I'm familiar with upgradeable contracts, but the Diamond Standard is a peculiar implementation.

An example project that shows how to import the library and use it in a base Counter or Greeter contract would be awesome.

**Comment by lampshade9909** [comment]
Check out the example contract I used in the Contracts directory: 

https://github.com/lampshade9909/DiamondSetter/tree/master/Contracts

There is one proxy contract and two logic contracts.  I deployed them on mainnet and called the set command which generated this tx to associate the logic contracts & function selectors with the proxy -> https://etherscan.io/tx/0x6fa3da9f80c1ee3f7d4128c866a3fd98c9f6a7d18e09980fe3f9303c141e20f2

**Comment by PaulRBerg** [comment]
Sorry, what I meant is separate the core contracts from the example.

Maybe worth publishing this as an npm package and have another repo refer to it by importing the core contracts from node_modules.

**Comment by lampshade9909** [comment]
Yep.  Ideally, the core components would be a package and the example project would be separate.  Unfortunately, I'm swamped and don't have the bandwidth this month.  Releasing this is better than releasing nothing.  At some point, I will make it into a python package.
